### DISCUSSION TITLE
My Bitnami LocalHost site is inaccessible after restored from backup

### DISCUSSION THREAD

**Discussion by remytoper** [community/community#34966] [New to GitHub]
I seriously need all your help
I am new to web designing and Zero knowledge about programming languages or other editing stuff. But, I successfully designed a website after watching Youtube videos. 
I used Bitnami Wordpress to  design the site on Localhost. I never had a paid host.
2days back I deleted few images from my home page by mistake. I couldn't undo the changes so I restored using the latest backed up file.
Restoring went up to 63% in progress ,held up over there and nothing happened so I escaped and refreshed the site.
After that entire Localhost site is inaccessible and showing I have a critical error. Nothing happens just the error message

Please... who has a good knowledge guide me through how to recover 😞....

**Comment by queenofcorgis** [comment]
Hi there - are you using GitHub Pages at all? 


If the answer is no and you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Bitnami or Localhost It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Flyerrebel** [comment]
I would download Google developers and they will teach u alot from articles and seminars.  Wht device are u using andriod or apple ?
